packages/react-core/src/components/Wizard/WizardFooter.tsx

```diff
@@ -3,13 +3,15 @@ import React from 'react';
 import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/Wizard/wizard';
 
-import { Button, ButtonVariant } from '../Button';
+import { Button, ButtonProps, ButtonVariant } from '../Button';
 import { isCustomWizardFooter, WizardStepType } from './types';
 
 /**
  * Hosts the standard structure of a footer with ties to the active step so that text for buttons can vary from step to step.
  */
 
+type FooterButtonProps = Omit<ButtonProps, 'children' | 'variant' | 'onClick'>;
```

Comment:
This needs to be placed before line 9. That comment currently renders a description of the WizardFooter component, and placing this type here removes that description on the site.